Master

CHDiaz opened the pull request:
-animation for movement and idle for Test-kun.
However it currently is not working...

CHDiaz committed:
Merge pull request #1 from simonayzman/master

Attempted Animation

CHDiaz committed:
Redoing Animation...

Had some sync problems...redoing animation state transition x_X
-Added code to TK_Ani.cs

CHDiaz committed:
Added

-Added Animations to states

CHDiaz committed:
Added State Transitions

-Redid all 50 million transitions...

CHDiaz committed:
Added Transition Conditions

-Added transitions conditions to all transitions (See Animation
tutorial)
-I forgot to mention that 2 commits back, I added 2 animation
parameters: "Tilde" and "Twalk"

CHDiaz committed:
Changed else if

-changed all "else if" in TK_Ani.cs to "if"

simonayzman commented on Darknet/Assets/Scripts/TK_Ani.cs:
```diff
@@ -2,13 +2,81 @@
+
+
+// Update is called once per frame
+void Update () {
```
I don't think your update function is checking conditions correctly. Check out the link that I provided in the Slack. The animation code the poster shows is more complicated; it checks to make sure that a specific key is pressed _and_ that you are _not_ in a certain state.

CHDiaz committed:
Added Scripts

-Possible Grid Scripts

CHDiaz committed:
Changed

-Removed unesscary camera components
-changed X rotation of camera to 0

CHDiaz committed:
Tested

-added "Gridd.cs"
-Tested grid method from

"http://answers.unity3d.com/questions/28223/what-is-the-best-way-to-make-modular-selectable-sq.html

CHDiaz committed:
Messed up states

CHDiaz committed:
Revert "Messed up states"

This reverts commit 9aa249f01516774dcb189b97663b79c3aa546513.

CHDiaz committed:
Pathfinding

-Added pathfinding files

CHDiaz committed:
Tried Navmesh

-Tried to use Navmesh

CHDiaz committed:
Removed Pathfinding 2D object

CHDiaz committed:
Started navmseh

= added first wall

CHDiaz committed:
Removed Extra Scenes

-Removed Overworld.unity
-Removed sl_Town-map.unity
-Removed Tutorial.unity

CHDiaz committed:
NavMesh-1

-Created several 3d world objects
-baked scene
-Moved All sprites to a "2D World" object

CHDiaz committed:
Agent created

-Created Agent.cs
-attached to agent
-removed "Pathfinding" folder (Just kept giving me errors and I wasnt
even using it)
-Moved 3D world and agent into better spot

CHDiaz committed:
Redid Test-kun animations

-Redid every single one of Test-kuns's animations
(Hopefully correctly this time)

CHDiaz committed:
Added FollowCube

-Added FollowCube.cs
Test-kun now follows the cube in the 3d world!

CHDiaz committed:
Redid states

-redid all states for Test-kun's animations
-Removed "TIdle"

CHDiaz committed:
Nav Mesh-2

-Edited FollowCube.cs to have animation transitions
-Pathfinding is working, with a few kinks.

CHDiaz committed:
Fixed dialg

-Fixed Diagonal movement and idle animations

CHDiaz committed:
Added Overworld

-Overworld created in same scene
-Added Grid-floor-1.png

CHDiaz committed:
Created Exit

-Created 3D objects (4) for exits
-Created 3D objects of Overworld
-added Navmesh to Overworld
-Created material for exits
-Created Transport.cs

CHDiaz committed:
Fixed walking glitch (moving before sprite change)

-Deleted Transport.cs

[Agent]
-NavMeshAgent Acceleration changed to 15
-NavMeshAgent Angular speed changed to 10
-Rigidbody mass changed to 6
-Rigidbody angular drag changed to 0
-Rigidbody gravity turned off

CHDiaz committed:
Changes

-Renamed "Town-Map" to "Darknet__Christian"
-Re baked Nav Mesh
-Added notwalkable tags to exit cubes

CHDiaz committed:
Several Changes

-Removed Exit.mat
-Added:
-Barrier.png
-Road.png
-Sign.png
-Tron_Floor5.png
-InBar.mat
-Changed 3D and 2D worlds to sphere shape
-Removed 2D and 3D world Overworld objects (For now)

CHDiaz committed:
Added

-Added MPanel.controller
-Added MPanel_Idle.anim
-Added TPanel.png
-Added game objects to 2D and 3D world for new marketplace design

CHDiaz committed:
Added

-Added APanel.controller
-Added  APanel_Idle.anim
-Added TPanel-2.png
-Added 2D objects for Arena in town

CHDiaz committed:
ReCali

-moved up all NPC to right layers, added 3D world objects for them
-Rebaked NavMesh
-Moved Teachers to Arena Game object

CHDiaz committed:
Messed around with Navmesh

CHDiaz committed:
Added New Map

-Added Sprites for New maps
-Added 3D world for new maps
-Rebaked navmesh

CHDiaz committed:
Finall Map

-Added the final map
-Rebaked NavMesh

CHDiaz committed:
Attempted Transition

-Added scripts for transisition
-Added 3D world panel objects, cube for testing
